### ISSUE TITLE
Rating Field

### ISSUE THREAD

**Issue by MountainAsh13** [giowck/symphytum#67]
Wondering if it would be possible to create a rating field type.  Would show a basic star icon depending on what was entered.  Should allow user to pick upper limit of rating, i.e. 5 stars, 10 stars, etc.

Thanks!

**Comment by joshirio** [comment]
Yes, definitively an useful field type, requires a custom widget though. Qt's docs may turn helpful for the part concerning the TableView: https://doc.qt.io/qt-5/qtwidgets-itemviews-stardelegate-example.html

**Comment by MountainAsh13** [comment]
Thanks for the quick reply!  Apologies if this is a dumb question, but is this plugin something that end users are able to install (and if so, are there instructions available) or would this be something earmarked for a future version of Symphytum?

**Comment by joshirio** [comment]
No the link above is just an example implementation where I could base future work on. Once the filed type is implemented it will be included as all other fields.
